DOCS.md

```diff
@@ -10,6 +10,9 @@ The following parameters are used to configure the plugin:
 * **recipient** - alternatively you can send it to a specific user
 * **username** - choose the username this integration will post as
 * **template** - overwrite the default message template
+* **image** - A valid URL to an image file that will be displayed inside a message attachment
```

Comment:
just to provide more explanation. The field in the yaml (`image_url`) is converted to all capital letters (`IMAGE_URL`) and then prefixed with `PLUGIN_` (`PLUGIN_IMAGE_URL`). This is how the value from the yaml is provided to the plugin as an environment variable.